vhdbuilder/packer/init-variables.sh

```diff
@@ -83,6 +83,31 @@ else
 	echo "Skipping SIG check for $MODE"
 fi
 
+# considerations to also add the windows support here instead of an extra script to initialize windows variables:
+# 1. we can demonstrate the whole user defined parameters all at once
+# 2. help us keep in mind that changes of this variables will influnce both windows and linux VHD building
+
+# windows image sku and windows image version are recorded in code instead of pipeline variables
+# because a pr gives a better chance to take a review of the version changes.
+WINDOWS_IMAGE_SKU=""
+WINDOWS_IMAGE_VERSION=""
+if [ ! -z "${WINDOWS_SKU}" ]; then
+	case "${WINDOWS_SKU}" in
+	"2019") 
+		WINDOWS_IMAGE_SKU="2019-Datacenter-Core-smalldisk"
+		WINDOWS_IMAGE_VERSION="17763.1282.2006061952"
+		;;
+	"2004")
```

Comment:
windows 2004 is not fully supported in this PR yet, changes like pause image are not included in this PR.